lake_viewer.py

```diff
@@ -1,70 +1,58 @@
 from pyiceberg import catalog
 from pyiceberg.expressions import AlwaysTrue
 from pyiceberg.catalog import Identifier
-from typing import List, Iterator, Union
-from ast import literal_eval
+from typing import List, Union
 import streamlit as st
 import pandas as pd
-import datetime
 import os
 import pyarrow as pa
 import pyarrow.ipc as ipc
-import requests
-import json
-import time
-import uuid, urllib
 from streamlit.components.v1 import html
+import dotenv
+dotenv.load_dotenv(dotenv.find_dotenv(usecwd=True)) #Use current working directory to load .env file
 
 class LakeView():
 
     def __init__(self):        
         self.catalog = catalog.load_catalog("default", 
             **{
+                'uri': os.environ.get("PYICEBERG_CATALOG__DEFAULT__URI"),
+                'token': os.environ.get("PYICEBERG_CATALOG__DEFAULT__TOKEN"),
```

Comment:
do we really need these two additions? I have been running w/o them for a while.

Reply:
I made them only to be able to use locally a `.env` file and reading it with dotenv. Otherwise, didn't read them.
Can be removed if you want.